connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -287,16 +288,12 @@ object KustoWriter {
         val blobPath = blobUri + sas
         val blobSourceInfo = new BlobSourceInfo(blobPath, size)
 
-        val retry = Retry.of("Ingest to Kusto", this.retryConfig)
-        val f:CheckedFunction0[IngestionResult] = new CheckedFunction0[IngestionResult]() {
-          override def apply(): IngestionResult= {
-            KDSU.logInfo(myName, s"Queued blob for ingestion in partition $partitionId for requestId: '$requestId}")
-            ingestClient.ingestFromBlob(blobSourceInfo, props)
-          }
-        }
-
-        val retryExecute: CheckedFunction0[IngestionResult] = Retry.decorateCheckedSupplier(retry, f)
-        if (transactional) partitionsResults.add(PartitionResult(retryExecute.apply,partitionId))
+        if (transactional) partitionsResults.add(
```

Comment:
I think this would be easier to read formatted as:
if (transactional) { // Even though body is technically 1 line
   // Remainder
}